QuasarHelios2015/src/org/team1540/quasarhelios/Elevator.java

```diff
@@ -20,20 +24,22 @@
     private static final ExtendedMotor winchCAN = Igneous.makeCANTalon(0);
     private static final FloatOutput winch = winchCAN.asMode(ExtendedMotor.OutputControlMode.VOLTAGE_FRACTIONAL);
 
-    private static final BooleanStatus raising = new BooleanStatus();
-    private static final BooleanStatus lowering = new BooleanStatus();
-    private static final BooleanStatus goingToMiddle = new BooleanStatus();
+    private static final BooleanStatus raising = new BooleanStatus(false);
+    private static final BooleanStatus lowering = new BooleanStatus(false);
+    private static final BooleanStatus goingToMiddle = new BooleanStatus(false);
+    
+    private static final BooleanStatus overrideEnabled = new BooleanStatus(false);
+    private static final FloatStatus overrideValue = new FloatStatus(0.0f);
```

Comment:
This needs to be set from somewhere. Maybe from a joystick on the copilot's joystick?